src/3d/qgsrubberband3d.h

```diff
@@ -116,49 +131,119 @@ class _3D_EXPORT QgsRubberBand3D
      */
     MarkerType markerType() const;
 
+    /**
+     * Sets the marker outline style
+     * \note Default value is solid line
+     * \since QGIS 3.44
+     */
+    void setMarkerOutlineStyle( Qt::PenStyle style );
+
+    /**
+     * Returns current marker outline style used by rubberband.
+     * \since QGIS 3.44
+     */
+    Qt::PenStyle markerOutlineStyle() const;
+
+    /**
+     * Sets whether to show markers on vertices
+     * \since QGIS 3.44
+     */
+    void setMarkerEnabled( bool enable );
+
+    /**
+     * Returns whether markers are being shown on vertices
+     * \since QGIS 3.44
+     */
+    bool markerEnabled() const;
```

Comment:
1. we have singular "markerEnabled" but plural "edgesEnabled", 2. in Qt APIs returning bool, it is common to name them isBlahBlah() or hasBlahBlah()

Here I would suggest:  hasMarkersEnabled(), hasEdgesEnabled(), hasFillEnabled()